src/path/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Path(models.Model):
+    start_location_id = models.TextField()
+    start_location_name = models.TextField()
+    end_location = models.TextField()
```

Comment:
`end_location_id` for consistency, also same question about being an integer

Reply:
omg nice catch